src/App.scss

```diff
@@ -6,31 +6,31 @@ iframe {
   text-align: center;
 }
 
-.PostInfo {
-  margin: 10px auto;
-  width: 90%;
-  border: 1px solid #000;
-  border-radius: 8px;
-  background-color: lightgray;
-  padding: 1em;
+// .PostInfo {
+//   margin: 10px auto;
+//   width: 90%;
+//   border: 1px solid #000;
+//   border-radius: 8px;
+//   background-color: lightgray;
+//   padding: 1em;
 
-  &__title {
-    margin: 0;
-  }
+//   &__title {
+//     margin: 0;
+//   }
 
-  &__header {
-    margin-bottom: 1em;
-  }
-}
+//   &__header {
+//     margin-bottom: 1em;
+//   }
+// }
 
-.UserInfo {
-  font-weight: bold;
-}
+// .UserInfo {
+//   font-weight: bold;
+// }
 
-.CommentList {
-  display: flex;
-  flex-direction: column;
-  gap: 0.7em;
-  background-color: #eee;
-  padding: 1em;
-}
+// .CommentList {
+//   display: flex;
+//   flex-direction: column;
+//   gap: 0.7em;
+//   background-color: #eee;
+//   padding: 1em;
+// }
```

Comment:
This block of styles is commented out. If these styles are not needed, consider removing them to clean up the file. If they are intended for future use, ensure they are correctly implemented and uncommented when necessary.